frontend/src/reusable/urls.js

```diff
@@ -0,0 +1,3 @@
+const BASE_URL = 'http://localhost:8080';
```

Comment:
This url should be the one from your deployed backend in Heroku.

Reply:
This is why nothing is working 😭